EDDiscovery/PanelAndPopOuts.cs

```diff
@@ -59,6 +59,7 @@ public enum PanelIDs        // id's.. used in tab controls, and in button pop ou
             Expedition,             // 27
             Trilateration,          // 28
             Settings,               // 29
+            ScanGrid,
```

Comment:
Please add the id (30) for troubleshooting scenarios.

Reply:
Yep!

Reply:
Done.